srcpkgs/toot/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'toot'
 pkgname=toot
-version=0.27.0
-revision=4
+version=0.28.1
+revision=5
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-BeautifulSoup4 python3-requests python3-setuptools python3-wcwidth
  python3-urwid"
 short_desc="Mastodon CLI client"
-maintainer="Nathan Owens <ndowens04@gmail.com>"
+maintainer="Jon Levin <jon@jefferiestube.net>"
```

Comment:
Don't take maintainer-ship of packages unless its Orphaned or you've asked the current maintainer. @ndowens